csv_export_base/models/csv_export_base.py

```diff
@@ -36,7 +36,12 @@ def _default_filename(self):
     start_date = fields.Date(string="Start Date", required=True)
     end_date = fields.Date(string="End Date", required=True)
 
+    def get_recordset(self):
+        """override this function for more complex record selection"""
+        return self.env[self._connector_model].search(self.get_domain())
+
     def get_domain(self):
+        """override this method to use simple domain on _model"""
         raise NotImplementedError
```

Comment:
as it no more called maybe it's better to remove it from base ?

Reply:
it is still called in other implementations :-)